roles/ovsconfig/templates/ifcfg-interface.j2

```diff
@@ -0,0 +1,6 @@
+BOOTPROTO=none
+DEVICE={{item}}
+ONBOOT=yes
+MTU=9000
+MASTER=bond0
+SLAVE=yes
```

Comment:
Minor but good if we can match our public docs, aka re-arrange the order of them.

DEVICE=eth0
ONBOOT=yes
BOOTPROTO=none
MTU=9000
MASTER=bond0
SLAVE=yes